lms/config/__init__.py

```diff
@@ -25,6 +25,12 @@ def configure(settings):
         "hashed_pw": sg.get("HASHED_PW"),
         "salt": sg.get("SALT"),
         "username": sg.get("USERNAME"),
+        # The secret string that's used to sign the session cookie.
+        # This needs to be a 64-byte, securely generated random string.
+        # For example you can generate one using Python 3 on the command line
+        # like this:
+        #     python3 -c 'import secrets; print(secrets.token_hex(nbytes=64))'
+        "cookie_signing_secret": sg.get("COOKIE_SIGNING_SECRET", required=True),
```

Comment:
"Cookie" is perhaps ambiguous here given that we're about to add cookies for other purposes. `SESSION_COOKIE_SECRET` perhaps?

On a related note, I see that h has a single env var for the secret, but uses that together with a unique salt per secret usage to derive a per-usage key. This has the advantage of reducing the number of env vars that need to be configured while respecting the advice in the Pyramid docs not to reuse secrets. The implementation in h adds a dependency on an external library, but a newer implementation could use functions from [hashlib](https://docs.python.org/3/library/hashlib.html#key-derivation).

Reply:
Renamed to `session_cookie_secret`.